ansible/recover_server.py

```diff
@@ -175,7 +188,7 @@ def _print_summary(jobs):
         if not skip_cleanup:
             if jobs[0].failed_task is not None:
                 output.append('Server %s cleanup failed, skip recovery.' % server)
-            jobs = jobs[1:]
+            jobs = jobs[2:]
```

Comment:
Are we not printing a summary for start-vms?

Reply:
Agreed, it would be nicer if `start-vms` could print result summary like `cleanup` does in from line#188 to line#190.

Reply:
Updated. Thanks!